_unused_topics/monitoring-monitoring-overview.adoc

```diff
@@ -38,7 +38,7 @@ All the components of the monitoring stack are monitored by the stack. Additiona
 * kube-controller-manager-operator
 * kube-scheduler
 * kubelet
-* monitor-sdn
+* monitor-ovn-kubernetes
```

Comment:
@juzhao Could you double confirm this changes is correct on ovn-kubernetes cluster?

Reply:
@zhaozhanqi the change is right, but not sure why still update for the `unused` doc?

Reply:
+1